helpers/guilds/getPruneCount.ts

```diff
@@ -3,23 +3,16 @@ import type { Bot } from "../../bot.ts";
 /** Check how many members would be removed from the server in a prune operation. Requires the KICK_MEMBERS permission */
 export async function getPruneCount(bot: Bot, guildId: bigint, options?: GetGuildPruneCountQuery) {
   if (options?.days && options.days < 1) throw new Error(bot.constants.Errors.PRUNE_MIN_DAYS);
-  if (options?.days && options.days > 30) {
-    throw new Error(bot.constants.Errors.PRUNE_MAX_DAYS);
-  }
+  if (options?.days && options.days > 30) throw new Error(bot.constants.Errors.PRUNE_MAX_DAYS);
 
-  let url = bot.constants.endpoints.GUILD_PRUNE(guildId);
-
-  if (options) {
-    url += "?";
-
-    if (options.days) url += `days=${options.days}`;
-    if (options.includeRoles) url += `&include_roles=${options.includeRoles}`;
-  }
+  const queries = [];
+  if (options?.days) queries.push(`days=${options.days}`);
+  if (options?.includeRoles) queries.push(`include_roles=${options.includeRoles}`);
```

Comment:
```suggestion
  let url = bot.constants.endpoints.GUILD_PRUNE(guildId);

  if (options) {
    url += "?";

    if (options.days) url += `days=${options.days}`;
    if (options.includeRoles) url += `&include_roles=${options.includeRoles}`;
  }
```

Reply:
@itohatweb
Is url? valid? (everything is optional)
Is url?&after valid? (everything is optional)